Mathlib/LinearAlgebra/Projection.lean

```diff
@@ -109,24 +109,20 @@ theorem coe_prodEquivOfIsCompl (h : IsCompl p q) :
 theorem coe_prodEquivOfIsCompl' (h : IsCompl p q) (x : p × q) :
     prodEquivOfIsCompl p q h x = x.1 + x.2 := rfl
 
-@[simp]
 theorem prodEquivOfIsCompl_symm_apply_left (h : IsCompl p q) (x : p) :
```

Comment:
sorry if this was already discussed, but can you please explain why the `simp` attribute is being removed from these? In my mind these are good `simp` lemmas, but I'm open to being wrong.